app/src/main/java/tttTDD/Java/ConsoleClient.java

```diff
@@ -144,13 +144,26 @@ private String formatCellDisplay(String cell, String maxInput) {
         StringBuilder formattedCell = new StringBuilder();
 
         if (cell.length() <= maxInput.length()) {
-            StringBuilder spaceAlignment = new StringBuilder();
 
-            IntStream.range(0, maxInput.length() - cell.length()).forEach(i -> spaceAlignment.append(" "));
-            formattedCell.append(spaceAlignment);
+            IntStream.range(0, maxInput.length() - cell.length()).forEach(i -> formattedCell.append(" "));
             formattedCell.append(cell);
         }
 
         return formattedCell.toString();
     }
+
+    private int playerInput() {
+        int value;
+
+        try {
+            value = input.nextInt();
```

Comment:
I wonder if setting the variable here is also necessary. Could you just `return input.nextInt();` here also? 

I think it should still catch the exception and if you re-call this method, it should work? Not positive, but it feels like this _should_ work